.github/workflows/self-test.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       run: |
         pyenv="py$(echo "${{ matrix.python-version }}" | tr -d .)"
         cd Release-Specs
-        tox -e "test,${pyenv},flake8,pylint" -- --doctest-modules --self-test \
+        tox -e "test,flake8,pylint" -- --doctest-modules --self-test \
```

Comment:
Had to piggy-back this, to make the tests runnable again. There seems to be some regression with tox at the moment, but pinning to a specific python version does not make much sense with the release tests anyways.